conngen/conngenmodule.cpp

```diff
@@ -68,17 +67,16 @@ ConnectionGeneratorModule::init( SLIInterpreter* i )
   ConnectionGeneratorType.setdefaultaction( SLIInterpreter::datatypefunction );
 
   // Register the user functions of the connection generator interface
```

Comment:
Comment does not match the name of the function

Reply:
I guess we can live with this. SLI's function names are confusing at times and especially to those new to the code.